### PR TITLE
Adding OWNERS file for event-bus and serverless components

### PR DESCRIPTION
**Pull request by anishj0shi**
w.r.t #5605 

Adding Owner alias for event-bus and serverless components and OWNERS file for respective components

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -31,3 +31,31 @@ aliases:
...
+    - @lilitgh
+    - @k15r
+    - @nachtmaar
+    - @anishj0shi
+    - @montaro
+    - @marcobebway
+    - @radufa
+    - @sayanh
+    - @antoineco
+  skydiving-tunas
```
Comment (k15r) [inline OWNERS_ALIASES]: twice twice?

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -31,3 +31,31 @@ aliases:
     - dariadomagala
     - akucharska
     - sjanota
+  skydiving-tunas
```
Comment (k15r) [inline OWNERS_ALIASES]:
```suggestion
  skydiving-tunas-approvers:
```

File: OWNERS_ALIASES (hunk 1 above)
Comment (k15r) [reply OWNERS_ALIASES]: i see the other teams differentiate between reviewers and approvers. Shall we do the same even if they would be the same initially?
```suggestion
  skydiving-tunas-reviewers:
```


if we decide to do so all the OWNERS files need to be updated

**Comment by anishj0shi** [comment]
@k15r @antoineco Please review it again.

**Pull request merged**